src/test/java/com/alkemy/ong/CommentControllerFunctionalTest.java

```diff
@@ -0,0 +1,291 @@
+                .build();
+        entity = new HttpEntity(null, headers);
+        // When
+        ResponseEntity<ErrorDTO> response = testRestTemplate.exchange(
+                endpointUrl,
+                HttpMethod.GET,
+                entity,
+                new ParameterizedTypeReference<>() {
+                },
+                Map.of("id", "5")
+        );
+        //Then
+        assertEquals(404, response.getStatusCode().value());
+    }
+
+    @Test
+    void testDeleteCommentsById_ShouldReturnResponseOk() {
+        //Given
+        String endpointUrl = commentControllerUrl + "/comments/{id}";
+        Long id = createdComment();
+        HttpHeaders headers = new HeaderBuilder()
+                .withValidToken("admin1@gmail.com", 3600L)
+                .build();
+        entity = new HttpEntity(null, headers);
+        // When
+        ResponseEntity<CommentDTO> response = testRestTemplate.exchange(
+                endpointUrl,
+                HttpMethod.DELETE,
+                entity,
+                new ParameterizedTypeReference<>() {
+                },
+                Map.of("id", id)
+        );
+        //Then
+        assertEquals(200, response.getStatusCode().value());
+    }
+
+    @Test
+    void testDeleteCommentsById_shouldReturnErrorDTO() {
+        //Given
+        String endpointUrl = commentControllerUrl + "/comments/{id}";
+        HttpHeaders headers = new HeaderBuilder()
+                .withValidToken("admin1@gmail.com", 3600L)
+                .build();
+        entity = new HttpEntity(null, headers);
+        // When
+        ResponseEntity<ErrorDTO> response = testRestTemplate.exchange(
+                endpointUrl,
+                HttpMethod.DELETE,
+                entity,
+                new ParameterizedTypeReference<>() {
+                },
+                Map.of("id", "5")
+        );
+        //Then
+        assertEquals(HttpStatus.NOT_FOUND, response.getBody().getCode());
+    }
+
+    @Test
+    void testCreateComments_ShouldReturnResponseCreated() {
+        //Given
+        CommentCreationDTO commentCreationDTO = CommentCreationDTO.builder()
+                .body("Cuerpo del commentario")
+                .newsId(idNews)
+                .build();
+        String endpointUrl = commentControllerUrl + "/comments";
+        HttpHeaders headers = new HeaderBuilder()
+                .withValidToken("admin1@gmail.com", 3600L)
+                .build();
+        entity = new HttpEntity(commentCreationDTO, headers);
+        //When
+        ResponseEntity<CommentDTO> response = testRestTemplate.exchange(
+                endpointUrl,
+                HttpMethod.POST,
+                entity,
+                new ParameterizedTypeReference<>() {
+                },
+                Map.of()
+        );
+        //Then
+        assertEquals(201, response.getStatusCode().value());
+    }
+
+    @Test
+    void testCreateComments_shouldReturnErrorDto() {
```

Comment:
Si bien está bien decir que debería venir un ErrorDto, sería mejor decir qué es lo que va a venir. Al estilo `testCreateComment_shouldReturnBadRequest`.
También se suele poner mas específico en el nombre del test: `testCreateComment_whenBodyIsBlank_shouldReturnBadRequest` o ponerle el primero que dije y agregarle la annotation `@DisplayName("Test create comment with blank body, should return bad request")`.